test/widgets/message_list_test.dart

```diff
@@ -1096,6 +1097,79 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+
+    group('action sheet visual feedback', () {
+      late Message message;
+
+      setUp(() {
+        // Create a basic message for testing
+        message = eg.streamMessage();
+      });
+
+      // Helper function to find DecoratedBox and get its color
+      Color? getBackgroundColor(WidgetTester tester) {
+        final decoratedBox = tester.widget<DecoratedBox>(
+          find.descendant(
+            of: find.byType(MessageWithPossibleSender),
+            matching: find.byType(DecoratedBox),
+          ),
+        );
+        return (decoratedBox.decoration as BoxDecoration).color;
+      }
+
+      Future<void> buildTestWidget(WidgetTester tester) async {
+        await setupMessageListPage(
+          tester,
+          messages: [message],
+        );
+      }
+
+      testWidgets('starts with transparent background', (tester) async {
+        await buildTestWidget(tester);
+
+        expect(
+          getBackgroundColor(tester),
+          equals(Colors.transparent),
+          reason: 'Message should start with transparent background',
+        );
+      });
+
+      testWidgets('shows tint color when long pressed', (tester) async {
+        await buildTestWidget(tester);
+
+        // Trigger long press
+        await tester.longPress(find.byType(MessageWithPossibleSender));
```

Comment:
Another comment that just restates what's clearly expressed in the code 🙂; let's remove it